README.md

```diff
@@ -1,2 +1,31 @@
-# CSCK541---EOM---Group-A
-End Of Module Assignment
+**End of Module Assignment - Group A**
+------ 
+
+This project builds a simple client/server network for the University of Liverpool's Software Development in Practice module.
+It includes an application to create a dictionary, serialisation it, and send to a server.
+
+
+**How do I get started?**
+------
+Install the necessary packages that can be found in SECTION
+
```

Comment:
Can add under this section
-Dependencies: Python version, Python IDE platform
-Installing command code: Download the repository or clone it using git clone ....
-Executing program: 
Open the command prompt inside the folder
xxx.py
yyy.py